pkgs/development/python-modules/mne/default.nix

```diff
@@ -18,26 +18,38 @@
   h5io,
   pymatreader,
   pythonOlder,
+  procps,
+  optipng,
 }:
 
 buildPythonPackage rec {
-  pname = "mne-python";
-  version = "1.9.0";
+  pname = "mne";
+  # https://github.com/mne-tools/mne-python/pull/13049 is required to build, it does not apply if fetchpatch'ed
+  stableVersion = "1.9.0";
+  version = "1.9.0-unstable-2025-05-01";
```

Comment:
nit: `version = ${stableVersion}-unstable-2025-05-01` seems slightly better

Reply:
i think that might break the find-replace logic of nix-update

Reply:
Makes sense, and not important anyway